Assets/Mirror/Core/NetworkBehaviour.cs

```diff
@@ -53,6 +66,24 @@ public abstract class NetworkBehaviour : MonoBehaviour
         [Obsolete(".hasAuthority was renamed to .isOwned. This is easier to understand and prepares for SyncDirection, where there is a difference betwen isOwned and authority.")] // 2022-10-13
         public bool hasAuthority => isOwned;
 
+        /// <summary>authority is true if we are allowed to modify this component's state. On server, it's true if SyncDirection is SERVER_TO_CLIENT. On client, it's true if SyncDirection is CLIENT_TO_SERVER and(!) if this object is owned by the client.</summary>
+        // on the client: if owned and if clientAuthority sync direction
+        // on the server: if serverAuthority sync direction
+        //
+        // for example, NetworkTransform:
+        //   client may modify position if ClientAuthority mode and owned
+        //   server may modify position only if server authority
+        //
+        // note that in original Mirror, hasAuthority only meant 'isOwned'.
+        // there was no syncDirection to check.
+        //
+        // also note that this is a per-NetworkBehaviour flag.
+        // another component may not be client authoritative, etc.
+        public bool authority =>
```

Comment:
change to `stateAuthority` or `hasStateAuthority` for more clear naming?

Reply:
sounds a bit long.
but would be nice to use hasAuthority again some day.
for now it has different meanings, so I picked .authority :)